### PR TITLE
Feat/byoip tracking

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/byoip`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-8440
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js [hunk 1]
```diff
@@ -299,6 +305,19 @@ export default class AgoraIpOrderCtrl {
...
+      name: `${TRACKING_PREFIX}confirm_${get(
+        this.model,
+        'selectedService.type',
+      )}_${get(this.model.params, 'selectedOffer.planCode')}_${get(
+        this.model.params,
+        'selectedQuantity',
+        1,
+      )}_${get(this.model.params, 'selectedCountry.code', null)}`,
+      type: 'action',
+    });
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js:319]: May be we can improved like this:

```suggestion
    const { params } = this.model;
    const serviceType = params.selectedService.type;
    const offerPlanCode = params.selectedOffer.planCode;
    const quantity = params.selectedQuantity || 1;
    const countryCode = params.selectedCountry.code || null;

    this.atInternet.trackClick({
      name: `${TRACKING_PREFIX}confirm_${serviceType}_${offerPlanCode}_${quantity}_${countryCode}`,
      type: 'action',
    });
```

**Review by JeremyDec — APPROVED** [review]

**Review by radireddy — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js [hunk 2]
```diff
@@ -298,6 +304,16 @@ export default class AgoraIpOrderCtrl {
   }
 
   redirectToPaymentPage() {
+    const { params } = this.model;
+    const serviceType = params.selectedService.type;
+    const offerPlanCode = params.selectedOffer.planCode;
+    const quantity = params.selectedQuantity || 1;
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js:310]: If `selectedQuantity` can be 0, better to use `nullish coalescing`

File: packages/manager/apps/dedicated/client/app/ip/byoip/disclaimer/disclaimer.routing.js [hunk 3]
```diff
@@ -6,31 +6,28 @@ export default /* @ngInject */ ($stateProvider) => {
...
-      goToExpressOrder: /* @ngInject */ ($window, expressOrderUrl) => () => {
-        return $window.open(expressOrderUrl, '_blank');
+      goToExpressOrder: /* @ngInject */ ($window) => (url) => {
+        return $window.open(url, '_blank');
       },
       breadcrumb: () => null,
     },
     atInternet: {
-      ignore: true,
+      ignore: 'dedicated::ip::dashboard::bring-your-own-ip::confirmation',
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/byoip/disclaimer/disclaimer.routing.js:30]: **question:** Shouldn't this be `rename`?

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js (hunk 2 above)
Comment (varun257) [reply packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js:310]: It is always 1 or above

File: packages/manager/apps/dedicated/client/app/ip/byoip/disclaimer/disclaimer.routing.js (hunk 3 above)
Comment (varun257) [reply packages/manager/apps/dedicated/client/app/ip/byoip/disclaimer/disclaimer.routing.js:30]: Oh yes, i will take care of this in https://github.com/ovh/manager/pull/6404

**Pull request merged**